tests/data/specs/tlv_with_checksum.rflx

```diff
@@ -9,6 +9,7 @@ package TLV_With_Checksum is
          Tag      : Tag
             then Length
                if Tag = Msg_Data
+                  or Tag = Msg_Undefined
             then null
                if Tag = Msg_Error;
```

Comment:
What is the intention of this change?

Reply:
We have a refinement in another unit that will check for `Tag = Msg_Data`. The data field that refinement is applied to is only reachable through this path. So if we kept `Tag = Msg_Data` here the refinement condition would be always true and therefore rejected. Alternatively I could have changed the refinement condition to `True` but I wanted to avoid losing test feature coverage by simplifying the test.

Reply:
Sounds reasonable, but reverting that change didn't lead to a failing test for me.

```diff
diff --git a/tests/data/specs/tlv_with_checksum.rflx b/tests/data/specs/tlv_with_checksum.rflx
index 862b7a5db..ceec731bb 100644
--- a/tests/data/specs/tlv_with_checksum.rflx
+++ b/tests/data/specs/tlv_with_checksum.rflx
@@ -1,6 +1,6 @@
 package TLV_With_Checksum is
 
-   type Tag is (Msg_Undefined => 0, Msg_Data => 1, Msg_Error => 3) with Size => 8;
+   type Tag is (Msg_Data => 1, Msg_Error => 3) with Size => 8;
    type Length is range 0 .. 2 ** 16 - 1 with Size => 16;
    type Checksum is range 0 .. 2 ** 32 - 1 with Size => 32;
 
@@ -9,7 +9,6 @@ package TLV_With_Checksum is
          Tag      : Tag
             then Length
                if Tag = Msg_Data
-                  or Tag = Msg_Undefined
             then null
                if Tag = Msg_Error;
          Length   : Length
```

Reply:
Did you check
```
python -m pytest "tests/integration/specification_model_generator_test.py::test_feature_integration" -vv -s
```
?

This test fails when I revert this change. It is not executed by default when the python unit tests are run but it will make the compilation tests fail.

Reply:
I see. Let's keep the change.